.circleci/config.yml

```diff
@@ -49,6 +49,7 @@ jobs:
       - run:
           command: |
             sudo npm install -g jest
+            sed -i -e 's/export {/module.exports = {/' web/src/helpers/constants.js
```

Comment:
I couldn't get Jest to work without doing a lot of extra work (like creating a package for this one test file), so instead we hack it by turning this file back into CommonJS. It's not ideal, but I think it's an acceptable workaround for now :)